test/transactions/zap-withdraw.ts

```diff
@@ -0,0 +1,180 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import * as api from 'test/fixtures/api';
+import * as common from '@protocolink/common';
+import { expect } from 'chai';
+import { getBalance, getChainId, polygonTokens, snapshotAndRevertEach } from '@protocolink/test-helpers';
+import hre from 'hardhat';
+import * as logics from '@protocolink/logics';
+import * as utils from 'test/utils';
+
+describe('Transaction: Zap Withdraw', function () {
+  const marketId = logics.compoundv3.MarketId.USDC;
+
+  let chainId: number;
+  let user1: SignerWithAddress;
+  let user2: SignerWithAddress;
+  let service: logics.compoundv3.Service;
+  let cToken: common.Token;
+
+  before(async function () {
+    chainId = await getChainId();
+    user1 = await hre.ethers.getImpersonatedSigner('0x0fbeabcafcf817d47e10a7bcfc15ba194dbd4eef');
```

Comment:
Shall we consider modifying it to checksum addresses for better consistency?

Reply:
fixed